src/windows/video_widget.py

```diff
@@ -1265,8 +1273,12 @@ def keyFrameTransformTriggered(self, effect_id, clip_id):
 
     def regionTriggered(self, clip_id):
         """Handle the 'select region' signal when it's emitted"""
-        # Clear transform
-        self.region_enabled = bool(not clip_id)
+        if self and not clip_id:
+            # Clear transform
+            self.region_enabled = False
+        else:
+            self.region_enabled = True
+
```

Comment:
OK, the only issue here was that I got the test backwards. It should be `self.region_enabled = bool(clip_id)`. But it's very odd to be testing `self` in a Python method. `self` is the current class instance. It's implicit. I've always wondered what those tests were meant to accomplish.

Reply:
(The reason I got the test backwards was that it sets `region_enabled` to `False` if `not clip_id` is `True`. Just like in speech, double negatives in code are confusing.)